src/osemgrep/language_server/server/dune

```diff
@@ -5,7 +5,6 @@
  (libraries
    lsp
    lwt
-   lwt.unix
```

Comment:
maybe add a comment in this file like "Note that we're not dependong on lwt.unix here! This is good because XXX"